tests/functional/idempotency/test_idempotency.py

```diff
@@ -995,8 +994,7 @@ def test_default_no_raise_on_missing_idempotency_key(
     hashed_key = persistence_store._get_hashed_idempotency_key({})
 
     # THEN return the hash of None
-    expected_value = f"test-func.{function_name}#" + md5(json_serialize(None).encode()).hexdigest()
-    assert expected_value == hashed_key
+    assert hashed_key is None
```

Comment:
The behavior when the key is `None` changed, so I had to change this test. I would like your opinion here.